TEMPLATE.md

```diff
@@ -47,3 +47,23 @@ Below are the repository settings and branch protection rules which you will nee
 ## Require status check to pass before merging
 
 `TODO add image` - search for "Add me as branch protection" and just use that one.
+
+<br />
+
+## Automatic workflows triggered on pushing code to different branches
+
+- **Create a PR to merge code in main branch**
+        -> Generates a new semantic version.
```

Comment:
maybe this section could be indented ordered list so its easier to read/see steps ?